### PR TITLE
Openstack keycloak review

### PR DESCRIPTION
**Pull request by enolfc**
# Summary

Fix the OpenStack migration guide to enable both issuers at the same time thanks to Keycloak introspection being able to also handle MITREid tokens.

This should make life easier for sites and users.

---

**Related issue :**

### CODE REVIEW

**Comment by gwarf** [comment]
Oh Enol, you are too doing PRs from non up to date branches/forks? ;)

**Comment by enolfc** [comment]
> Oh Enol, you are too doing PRs from non up to date branches/forks? ;)

me too :) but have fixed it quickly

File: content/en/providers/cloud-compute/openstack/aai.md [hunk 1]
```diff
@@ -737,19 +740,29 @@ Similarly to the change from Demo to Production described
 [above](#moving-to-egi-check-in-production-instance), you will need to update
 the URLs on your configuration:
 
-1. Update the `remote-id` of the identity provider:
+1. Update the `remote-id` of the identity provider to also include the new
+   issuer:
 
    ```shell
-   $ openstack identity provider set --remote-id https://aai.egi.eu/auth/realms/egi egi.eu
+   $ openstack identity provider set --remote-id https://aai-dev.egi.eu/auth/realms/egi --remote-id https://aai-dev.egi.eu/oidc/ egi.eu
```
Comment (nikosev) [inline content/en/providers/cloud-compute/openstack/aai.md:747]: I suppose that the instruction refer to the production Check-in , so the command should be:

```
openstack identity provider set --remote-id https://aai.egi.eu/auth/realms/egi --remote-id https://aai.egi.eu/oidc/ egi.eu
```

File: content/en/providers/cloud-compute/openstack/aai.md [hunk 2]
```diff
@@ -737,19 +740,29 @@ Similarly to the change from Demo to Production described
...
    ```
 
-1. Update the `HTTP_OIDC_ISS` filter in your mappings, e.g.:
+1. Add the new issuer to the `HTTP_OIDC_ISS` filter in your mappings, keep both
+   the `https://aai-dev.egi.eu/oidc/` and
+   `https://aai-dev.egi.eu/auth/realms/egi` so users can still use their
```
Comment (nikosev) [inline content/en/providers/cloud-compute/openstack/aai.md:752]: same here (production URL): `https://aai.egi.eu/auth/realms/egi`

File: content/en/providers/cloud-compute/openstack/aai.md [hunk 3]
```diff
@@ -669,7 +669,7 @@ and Demo environments already using Keycloak since June 24th 2022 and Production
 environment expected to migrate during July 2022.
 
 A general guide on migration is available on
-[Check-in documentation](../../../check-in/sp/#client-migration-to-keycloak), in
+[Check-in documentation](../../check-in/sp/#client-migration-to-keycloak), in
```
Comment (nikosev) [inline content/en/providers/cloud-compute/openstack/aai.md:672]: Could you please revert this change? The link in the [preview](https://docs.egi.eu/documentation/469/providers/cloud-compute/openstack/aai/#client-migration-to-keycloak) is broken.
Here is a screenshot of the redirect when I hover the mouse above the `Check-in documentation` link:
Screen Shot 2022-07-06 at 12 13 38
CC: @gwarf

**Review by nikosev — CHANGES_REQUESTED** [review]
Please see the submitted comments

File: content/en/providers/cloud-compute/openstack/aai.md (hunk 1 above)
Comment (enolfc) [reply content/en/providers/cloud-compute/openstack/aai.md:747]: yup, copy pasted too much

Comment (enolfc) [inline content/en/providers/cloud-compute/openstack/aai.md:747]:
```suggestion
   $ openstack identity provider set --remote-id https://aai.egi.eu/auth/realms/egi --remote-id https://aai.egi.eu/oidc/ egi.eu
```

File: content/en/providers/cloud-compute/openstack/aai.md (hunk 2 above)
Comment (enolfc) [inline content/en/providers/cloud-compute/openstack/aai.md:752]:
```suggestion
   `https://aai.egi.eu/auth/realms/egi` so users can still use their
```

File: content/en/providers/cloud-compute/openstack/aai.md (hunk 3 above)
Comment (gwarf) [reply content/en/providers/cloud-compute/openstack/aai.md]: I more or less reverted the change, seems it's again a false error, not exactly sure what is happening here :/

Comment (gwarf) [reply content/en/providers/cloud-compute/openstack/aai.md]: Let's ignore the error and bypass it for the merge

**Review by nikosev — APPROVED** [review]
LGTM
